### PR TITLE
update Readme to reflect current project state

### PR DESCRIPTION
**Pull request by MichaelClifford**
## Related Issues and Dependencies

#34 

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Updated the readme to reflect the current state of the project, as well as added a number of links so that whether or not you interact with this content on github or operate-first you should be re-directed correctly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,21 +24,39 @@ The higher order goal to keep in mind for any tools developed here should be to
...
 tools and relevant data as accessible as possible to foster collaboration and contributions between data scientists and
-OpenShift engineers.
+DevOps engineers.
 
-## First Step/ Ways to get Involved:
+## Current Work
 
-**1 )** Exploratory data analysis of available data sources (gcsweb, TestGrid, Sippy) to understand the different data
-types that need to be addressed and identify what kind of ML approaches are possible.
+* Interactive and reproducible notebooks for the [entire project](https://github.com/aicoe-aiops/ocp-ci-analysis) are available at the [operate-first jupyterhub](https://jupyterhub-opf-jupyterhub.apps.cnv.massopen.cloud/hub/login)
```
Comment (oindrillac) [inline README.md:31]: should this be rephrased to `are available as an image on [jupyterhub](https://jupyterhub-opf-jupyterhub.apps.cnv.massopen.cloud/hub/login) deployed on the MOC cluster`

Comment (MichaelClifford) [reply README.md:31]: @oindrillac :+1:

**Review by oindrillac — APPROVED** [review]
LGTM

**Comment by MichaelClifford** [comment]
@durandom @tumido would either of you like to add an additional review to this PR before its merged :smile:

**Review by durandom — APPROVED** [review]

**Pull request merged**